sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -32,6 +32,8 @@ class GestureArenaEntry {
 class GestureArena {
```

Comment:
Add a comment on acceptGesture and rejectGesture that one of these will be called exactly once, whether you try to resolve or whether someone else tries to resolve. (Once could assume that e.g. if you try to resolve accepted, that we won't call acceptGesture(), it should be clear that we will.)